include/UIKit/UIKitTypes.h

```diff
@@ -48,7 +48,7 @@ typedef NS_ENUM(NSUInteger, NSWritingDirection) {
 };
 
 enum {
-    UITextAlignmentLeft,
+    UITextAlignmentLeft = 0,
     UITextAlignmentCenter,
```

Comment:
Perhaps NS should just be implemented in terms of UI?

---
In reply to: [85553544](https://github.com/Microsoft/WinObjC/pull/1254#discussion_r85553544) [](ancestors = 85553544)